Makefile

```diff
@@ -0,0 +1,12 @@
+PYTHONPATH := ./torch:$(PYTHONPATH)
+
+.PHONY: test lint setup dist
+
+test:
+	PYTHONPATH=$(PYTHONPATH) pytest tests/
+lint:
+	pylint --disable=E1101,W0719,C0202,R0801,W0613 src/
+setup:
+	python3 -m pip install -r requirements.txt -U
+dist/archive.tar.gz:
+	tar -czf module.tar.gz run.sh requirements.txt src
```

Comment:
This is the old run scrip, you'll want to package the binary ouput by pyinstaller instead
`tar -czvf dist/archive.tar.gz dist/main`